pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -283,8 +283,18 @@ type WorkflowSpec struct {
 	//Optional: Defaults to empty.
 	// +optional
 	PodDisruptionBudget *policyv1beta.PodDisruptionBudgetSpec `json:"podDisruptionBudget,omitempty" protobuf:"bytes,31,opt,name=podDisruptionBudget"`
+
+	// Shutdown will shutdown the workflow according to its ShutdownStrategy
+	Shutdown ShutdownStrategy `json:"shutdown,omitempty" protobuf:"bytes,32,opt,name=shutdown,casttype=ShutdownStrategy"`
```

Comment:
This should be under the status section.

Reply:
Why so? `Status` should be read-only and we have `Suspend` under `spec` which works similarly to this